anker/src/lib.rs

```diff
@@ -43,6 +43,10 @@ pub fn find_instance_address(anker_program_id: &Pubkey, solido_instance: &Pubkey
     Pubkey::find_program_address(&[solido_instance.as_ref()], anker_program_id)
 }
 
+// TODO(ruuda): Use this below.
```

Comment:
I started working on this, but it is going to be a pretty big and invasive change, so I split it off into a separate PR: https://github.com/ChorusOne/solido/pull/488